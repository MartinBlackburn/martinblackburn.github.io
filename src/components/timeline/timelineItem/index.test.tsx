// libraries
import { render } from "@testing-library/react";

// component under test
import TimelineItem from ".";

describe("TimelineItem component", () => {
    test("renders correctly", () => {
        const { asFragment } = render(<TimelineItem />);

        expect(asFragment()).toMatchSnapshot();
    });
});
