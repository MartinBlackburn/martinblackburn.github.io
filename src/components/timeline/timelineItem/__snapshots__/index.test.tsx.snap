// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimelineItem component renders correctly with children 1`] = `
<DocumentFragment>
  <div
    class="timelineItem"
  >
    <h2
      class="timelineItem__title"
    >
      some title
    </h2>
    <div
      class="timelineItem__date"
    >
      <time
        datetime="1986-02-11"
      >
        11 February 1986
      </time>
        
    </div>
    <div>
      test
    </div>
  </div>
</DocumentFragment>
`;

exports[`TimelineItem component renders correctly with dark mode 1`] = `
<DocumentFragment>
  <div
    class="timelineItem timelineItem--dark"
  >
    <h2
      class="timelineItem__title"
    >
      some title
    </h2>
    <div
      class="timelineItem__date"
    >
      <time
        datetime="1986-02-11"
      >
        11 February 1986
      </time>
        
    </div>
  </div>
</DocumentFragment>
`;

exports[`TimelineItem component renders correctly without dark mode 1`] = `
<DocumentFragment>
  <div
    class="timelineItem"
  >
    <h2
      class="timelineItem__title"
    >
      some title
    </h2>
    <div
      class="timelineItem__date"
    >
      <time
        datetime="1986-02-11"
      >
        11 February 1986
      </time>
        
    </div>
  </div>
</DocumentFragment>
`;
